### PR TITLE
Changing systemd CentOS7 init script

### PR DESCRIPTION
**Pull request by Karen09**
Hi

Nomad systemd script has failure which is depends that nomad service need to run more processes then just one. so please have a look and discuss in case of needed

Regards 
Karen

### CODE REVIEW

File: templates/nomad.systemd.erb [hunk 1]
```diff
@@ -1,18 +1,16 @@
 [Unit]
-Description=Nomad Agent
-Wants=basic.target
-After=basic.target network.target
+Description=Nomad agent
+Requires=network-online.target
+After=network-online.target
 
 [Service]
-User=<%= scope.lookupvar('nomad::user') %>
```
Comment (dudemcbacon) [inline templates/nomad.systemd.erb]: I'm not super familiar with systemd init scripts. Can you describe why removing this was necessary? Same with the line below.

File: manifests/config.pp [hunk 2]
```diff
@@ -34,6 +34,12 @@
         }
       }
       'systemd' : {
+        file { '/etc/sysconfig/nomad':
+          ensure  => directory,
```
Comment (dudemcbacon) [inline manifests/config.pp]: Can you fix the indentation on line 38-41? Puppetlint will complain about it otherwise.

**Comment by dudemcbacon** [comment]
Hi @Karen09, thanks for the fix! See my comments above.

File: templates/nomad.systemd.erb (hunk 1 above)
Comment (Karen09) [inline templates/nomad.systemd.erb]: Hi 
I am also not that good in systemd but when there is credentials about user and group systemd service do not work as well.
i have tried with user and group the same script . does not worked in my case.
Regards
Karen

File: manifests/config.pp (hunk 2 above)
Comment (Karen09) [inline manifests/config.pp]: hi 
i can't see any fail in indentation on line 38-41, can you give more details? it started with 2 space and 4 after class declaration. is it not the right way ?
Regards
Karen

**Comment by Karen09** [comment]
Hi
Thanks for collaboration and trust.

Regards
Karen

**Pull request merged**